pkg/controller/servicebindingrequest/binder_test.go

```diff
@@ -56,6 +56,9 @@ func TestBinderNew(t *testing.T) {
 		map[string]string{},
 	)
 
+	f.AddMockedUnstructuredSecretRV(name)
```

Comment:
It might not be required another method for this resource, as `AddMOckedUnstructuredSecret` returns a reference for the secret that can be modified to add the resource version:

```go
s := f.AddMockedUnstructuredSecret(name)
s.SetResourceVersion("112200")
```

Reply:
I tried this but then s.metadata.resourceVersion is not working for me.

Reply:
Adding another secret due to https://github.com/redhat-developer/service-binding-operator/pull/523#discussion_r452005133